src/app/shared/layout/header.component.ts

```diff
@@ -113,6 +113,7 @@ export class HeaderComponent implements OnInit {
   }
 
   goBack() {
+    this.errorHandlingService.clearError();
```

Comment:
This is only called when the user go backs by header, does it clear error if user goes back by Alt Left key or browser back?

Maybe it might be better to implement dismissing bars on electron or when changing pages

Reply:
hi @gycgabriel, thanks for your comment. Yes I realise now that when the user goes back using the browser it does not clear the bar. I was thinking of using the router from `@angular/router` to listen for route changes in the error handling component, then clear the error bar if the route changes. 

However, I ran into a cyclic dependancy issue that I'm currently unable to solve when I place the router in the constructor of the error handling component:
<img width="443" alt="image" src="https://user-images.githubusercontent.com/67156011/183018697-0a67360c-1c48-47ef-9f07-1e3e34caf897.png">

Would you or anyone else happen to know how to resolve this issue? Thank you!

Reply:
Currently, I've placed the logic to clear the error component in `app.module.ts`, and it now works whenever the user navigates away from the page. Of course, whether we merge would depend on the decision of the team if this behaviour is desirable. Do take a look at the code when convenient, thank you